locator/abstract_replication_strategy.hh

```diff
@@ -206,6 +206,8 @@ public:
 
     virtual tablet_replica_set get_replicas_for_reading(const token& search_token) const = 0;
 
+    virtual std::pair<dht::token, dht::token> get_token_range(const token& search_token) const = 0;
```

Comment:
We should document that this pair represents a range `[first, second]`, which is not obvious.